audit/src/main/resources/config/audit.yml

```diff
@@ -20,6 +20,8 @@ auditOnError: false
 # log level; by default set to info
 logLevelIsError: false
 
+# the format for outputting the timestamp, if not specify, will use long value
```

Comment:
lets also document here what would happen if the format that user entered is invalid.